lib/Core/Dependency.cpp

```diff
@@ -1488,6 +1488,11 @@ void Dependency::executeMemoryOperation(
     std::vector<ref<Expr> > &args, bool inBounds, bool symbolicExecutionError) {
   execute(instr, callHistory, args, symbolicExecutionError);
 #ifdef ENABLE_Z3
+  if (NoBoundCheck)
+	// No Bounds check is needed to be performed. So, we just return. This is
+	// important when the user wants to get the skeleton tree.
```

Comment:
This section is not properly indented: it has 3 extra spaces at each line in the if conditional's body, compared to original code.